### ISSUE TITLE
Mangle can't be disabled.

### ISSUE THREAD

**Issue by valera1401** [webpack-contrib/uglifyjs-webpack-plugin#329]
new UglifyJsPlugin({
                    uglifyOptions: {
                        parallel: true,
                        cache: true,
                        output: {
                            comments: false,
                        },
                        compress: {
                            // The Android SBG has problems parsing the output
                            // when these options are enabled
                            'collapse_vars': platform !== "android",
                            sequences: platform !== "android",
                            keep_classnames: true,
                            keep_fnames: true
                        },
                        mangle: false,
                        keep_classnames: true,
                        keep_fnames: true
                    }
                })

I've tried all available options but couldn't disable mangle. How can I do it ?

**Comment by evilebottnawi** [comment]
@valera1401 `uglifyOptions.mangle: false`

**Comment by valera1401** [comment]
Sorry but i don't understand your answer. Where should I do it ? I tried to set mangle (and only mangle) option to false but it didn't help.

**Comment by xiangwan** [comment]
same problem too. 

`
new UglifyPlugin({
        uglifyOptions: {
          compress: {
            // turn off flags with small gains to speed up minification
            arrows: false,
            collapse_vars: false, // 0.3kb
            comparisons: false,
            computed_props: false,
            hoist_funs: false,
            hoist_props: false,
            hoist_vars: false,
            inline: false,
            loops: false,
            negate_iife: false,
            properties: false,
            reduce_funcs: false,
            reduce_vars: false,
            switches: false,
            toplevel: false,
            typeofs: false,

            // a few flags with noticable gains/speed ratio
            // numbers based on out of the box vendor bundle
            booleans: true, // 0.7kb
            if_return: true, // 0.4kb
            sequences: true, // 0.7kb
            unused: true, // 2.3kb

            // required features to drop conditional branches
            conditionals: true,
            dead_code: true,
            evaluate: true
          },
          mangle: false
        },
        sourceMap: false,
        cache: true,
        parallel: true
      })
`

**Comment by kindofone** [comment]
I installed [**uglifyjs-3-webpack-plugin**](https://github.com/whtsky/uglifyjs-3-webpack-plugin):
```
npm i -D uglifyjs-3-webpack-plugin
```

Required it in my `webpack.config.js` instead of the built-in uglifyjs-webpack-plugin:
```JavaScript
const UglifyJsPlugin = require("uglifyjs-3-webpack-plugin");
```

Now `mangle: false` is working again! 🎉
